packages/lodestar/test/spec/allForks/forkChoice.ts

```diff
@@ -157,6 +157,11 @@ export function forkChoiceTest(fork: ForkName): void {
         timeout: 10000,
         // eslint-disable-next-line @typescript-eslint/no-empty-function
         expectFunc: () => {},
+        shouldSkip: (_testCase, name, _index) => {
+          const ignoreTestsWithKeywords = ["proposer_boost"];
```

Comment:
This skip will be removed with the PR:https://github.com/ChainSafe/lodestar/pull/3540, as part of strategy to update the spec version in easier manageable PRs